Documentation/install/upgrade.rst

```diff
@@ -209,35 +290,90 @@ Both files are dedicated to "\ |SCM_BRANCH|" for each Kubernetes version.
       $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/1.11/cilium-rbac.yaml
       $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/1.11/cilium-ds.yaml
 
+.. _upgrade_version:
 
-You can also substitute the desired Cilium version number for vX.Y.Z in the
-command below, but be aware that copy of the spec file stored in Kubernetes
-might run out-of-sync with the CLI flags, or options, specified by each Cilium
-version.
+Direct version upgrade
```

Comment:
I would add `(Not recommended for major or minor upgrades)`

Reply:
I might put this in a drop-down note? Otherwise this bracketed part would be duplicated in each of the 1.0, 1.1, 1.2-specific instructions links.